python3.6/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6-slim-jessie
+FROM python:3.6-slim-buster
```

Comment:
Why use the testing branch "buster" instead of the stable "stretch" ?

Reply:
Good question, the Debian release page (https://www.debian.org/releases/) states that `buster` is now the current stable while `stretch` is the old stable. We have chosen to go to `buster` release directly, to avoid switching again sooner than really required when using stretch.

Reply:
I checked the page good choice then. 👍 

It looks like bullseye is now testing and buster stable